src/test/java/seedu/tr4cker/testutil/TaskUtil.java

```diff
@@ -30,7 +30,7 @@ public static String getAddCommand(Task task) {
     public static String getTaskDetails(Task task) {
         StringBuilder sb = new StringBuilder();
         sb.append(PREFIX_NAME + task.getName().taskName + " ");
-        sb.append(PREFIX_DEADLINE + task.getDeadline().value + " ");
+        sb.append(PREFIX_DEADLINE + task.getDeadline().toString() + " ");
```

Comment:
would it be better to wrap the date time inside deadline with another class. so that we can just .dateTime instead of calling toString()

Reply:
I think using toString() is apt because what we want is a String representation of a Deadline.